src/config/KialiIcon.tsx

```diff
@@ -16,29 +16,40 @@ import {
   LockOpenIcon,
   BlueprintIcon,
   AngleDoubleUpIcon,
-  AngleDoubleDownIcon
+  AngleDoubleDownIcon,
+  BellIcon,
+  AngleDoubleLeftIcon,
+  AngleDoubleRightIcon
 } from '@patternfly/react-icons';
 import { style } from 'typestyle';
 
 const iconStyle = style({
   width: '10px'
 });
 
+// keep alphabetized
 export const KialiIcon = {
-  Info: () => <InfoAltIcon className={iconStyle} />,
-  Ok: () => <OkIcon className={iconStyle} color={PfColors.Green400} />,
-  Warning: () => <WarningTriangleIcon className={iconStyle} color={PfColors.Orange400} />,
-  Error: () => <ErrorCircleOIcon className={iconStyle} color={PfColors.Red100} />,
-  Unknown: () => <UnknownIcon className={iconStyle} />,
-  Topology: () => <TopologyIcon className={iconStyle} />,
-  Services: () => <ServiceIcon className={iconStyle} />,
-  Applications: () => <ApplicationsIcon className={iconStyle} />,
-  Workloads: () => <BundleIcon className={iconStyle} />,
-  VirtualService: () => <BlueprintIcon className={iconStyle} />,
-  CircuitBreaker: () => <BoltIcon className={iconStyle} />,
-  MissingSidecar: () => <CodeBranchIcon className={iconStyle} />,
-  MtlsLock: () => <LockIcon className={iconStyle} />,
-  MtlsUnlock: () => <LockOpenIcon className={iconStyle} />,
-  AngleDoubleUp: () => <AngleDoubleUpIcon className={iconStyle} />,
-  AngleDoubleDown: () => <AngleDoubleDownIcon className={iconStyle} />
+  AngleDoubleDown: (style?: string) => <AngleDoubleDownIcon className={style !== undefined ? style : iconStyle} />,
+  AngleDoubleLeft: (style?: string) => <AngleDoubleLeftIcon className={style !== undefined ? style : iconStyle} />,
+  AngleDoubleRight: (style?: string) => <AngleDoubleRightIcon className={style !== undefined ? style : iconStyle} />,
+  AngleDoubleUp: (style?: string) => <AngleDoubleUpIcon className={style !== undefined ? style : iconStyle} />,
+  Applications: (style?: string) => <ApplicationsIcon className={style !== undefined ? style : iconStyle} />,
+  Bell: (style?: string) => <BellIcon className={style !== undefined ? style : iconStyle} />,
+  CircuitBreaker: (style?: string) => <BoltIcon className={style !== undefined ? style : iconStyle} />,
+  Error: (style?: string) => (
+    <ErrorCircleOIcon className={style !== undefined ? style : iconStyle} color={PfColors.Danger} />
+  ),
+  Info: (style?: string) => <InfoAltIcon className={style !== undefined ? style : iconStyle} color={PfColors.Info} />,
```

Comment:
I see two options here:

1. Create an `IconProps` type here that has the `style?: string` attribute and use it like this:
```
interface IconProps {
  style?: string;
}

Info: (props: IconProps) => <InfoAltIcon className={props.style !== undefined ? props.style : iconStyle} color={PfColors.Info} />,
```

2. Use object destructuring
```
  Info: ({style}: {style?: string}) => <InfoAltIcon className={style !== undefined ? style : iconStyle} color={PfColors.Info} />,

```
Both options allow to use it as a React component `<KialiIcon.Info style="foo"/>" or `<KialiIcon.Info/>`

I recommend to use option 1), as we have several Icons and I would use `className` instead of `style` to avoid mixing both semantics. Also, we can refactor the whole `style !== undefined ? style : iconStyle` out of the components, by giving the Icons defaultProps.

I'll write a PR to your branch with my suggested changes, but feel free to update as you see fit.

Reply:
option 1 looks good to me, thanks for the pending PR...